src/vm/gcheaputilities.cpp

```diff
@@ -152,6 +152,69 @@ void StashKeywordAndLevel(bool isPublicProvider, GCEventKeyword keywords, GCEven
     }
 }
 
+#ifdef FEATURE_STANDALONE_GC
+HMODULE LoadStandaloneGc(LPCWSTR libFileName)
+{
+    LIMITED_METHOD_CONTRACT;
+
+    //
+    // Look for the standalone GC module next to the hosting binary
+    //
+
+    PathString libPath;
+    DWORD result = WszGetModuleFileName(nullptr, libPath);
```

Comment:
Is the idea that the standalone GC library is always going to be next to CoreCLR.dll ? If yes, then this should be better calling GetModuleFileName for CoreCLR.dll; not for the module that started the process.

Reply:
Sure I can do that.